cmd/kubelet-plugin/common.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"log"
+)
+
+type DeviceHandler interface {
+	GetDeviceIOMMUGroup(basepath string, pciAddress string) (string, error)
+	GetDeviceDriver(basepath string, pciAddress string) (string, error)
+	GetDeviceNumaNode(basepath string, pciAddress string) (numaNode int)
+	GetDevicePCIID(basepath string, pciAddress string) (string, error)
+}
+
+type DeviceUtilsHandler struct{}
```

Comment:
Do we need to expose this type or can we simply use the interface? Usually, the type is private to the package and there is a costructor that returns the interface:
```golang
func NewDeviceHandler() DeviceHandler {
    return &deviceUtilsHandler{}
}
```

Reply:
yes exposing this is not required , I will add the  NewDeviceHandler()